model_info/enwiki.drafttopic.md

```diff
@@ -15,611 +15,947 @@ Model Information:
+		Geography.Regions.Africa.Africa*                0.092         0.008
+		Geography.Regions.Africa.Central Africa         0             0
+		Geography.Regions.Africa.Eastern Africa         0.018         0
+		Geography.Regions.Africa.Northern Africa        0.02          0.001
+		Geography.Regions.Africa.Southern Africa        0.021         0.001
+		Geography.Regions.Africa.Western Africa         0.019         0.001
+		Geography.Regions.Americas.Central America      0.021         0.003
+		Geography.Regions.Americas.North America        0.121         0.064
+		Geography.Regions.Americas.South America        0.025         0.006
+		Geography.Regions.Asia.Asia*                    0.177         0.046
+		Geography.Regions.Asia.Central Asia             0.019         0.001
+		Geography.Regions.Asia.East Asia                0.043         0.011
+		Geography.Regions.Asia.North Asia               0.022         0.001
+		Geography.Regions.Asia.South Asia               0.039         0.015
+		Geography.Regions.Asia.Southeast Asia           0.027         0.006
+		Geography.Regions.Asia.West Asia                0.036         0.011
+		Geography.Regions.Europe.Eastern Europe         0.05          0.013
+		Geography.Regions.Europe.Europe*                0.199         0.076
+		Geography.Regions.Europe.Northern Europe        0.068         0.031
+		Geography.Regions.Europe.Southern Europe        0.037         0.013
+		Geography.Regions.Europe.Western Europe         0.051         0.019
+		Geography.Regions.Oceania                       0.041         0.015
+		History and Society.Business and economics      0.054         0.01
+		History and Society.Education                   0.035         0.007
+		History and Society.History                     0.051         0.011
+		History and Society.Military and warfare        0.064         0.014
+		History and Society.Politics and government     0.074         0.028
+		History and Society.Society                     0.064         0.013
+		History and Society.Transportation              0.057         0.015
+		STEM.Biology                                    0.046         0.034
+		STEM.Chemistry                                  0.021         0.002
+		STEM.Computing                                  0.032         0.003
+		STEM.Earth and environment                      0.026         0.005
+		STEM.Engineering                                0.036         0.005
+		STEM.Libraries & Information                    0.018         0.001
+		STEM.Mathematics                                0.017         0
+		STEM.Medicine & Health                          0.027         0.006
+		STEM.Physics                                    0.019         0.001
+		STEM.STEM*                                      0.26          0.069
+		STEM.Space                                      0.022         0.006
+		STEM.Technology                                 0.057         0.005
+	match_rate (micro=0.051, macro=0.016):
```

Comment:
the match_rate seems to have dropped a bit, not sure if it's an issue though.

Reply:
This is one that got hit by "population rates"